modin/pandas/test/test_io.py

```diff
@@ -973,6 +1028,10 @@ def test_to_csv(self, request, header, mode):
             mode=mode,
         )
 
+    @pytest.mark.skipif(
+        Backend.get() == "Omnisci",
+        reason="to_csv is not implemented with OmniSci backend yet",
```

Comment:
maybe submit a tracker and reference it here?

Reply:
Fixed.